content/en/references/internal-endpoints.md

```diff
@@ -15,13 +15,14 @@ LocalStack provides several internal endpoints for various local AWS services an
 The API path for the LocalStack internal resources is `/_localstack`. The following endpoints are available:
 
 | Endpoint | Description |
-| ---------------------------------| --------------------------------------------------------------------------- |
-| `/_localstack/health`| To check the available and running AWS services in LocalStack. You can use the endpoint to restart the LocalStack services.  |
-| `/_localstack/plugins` | Shows the [Plux plugins ](https://github.com/localstack/localstack/blob/master/docs/localstack-concepts/README.md#plugins) information in LocalStack. |
-| `/_localstack/init`| Shows the initialization status after setting up [Init hooks](https://docs.localstack.cloud/references/init-hooks/). |
+| ------------------------------------ | --------------------------------------------------------------------------- |
+| `/_localstack/health`                | To check the available and running AWS services in LocalStack. You can use the endpoint to restart the LocalStack services.  |
+| `/_localstack/plugins`               | Shows the [Plux plugins ](https://github.com/localstack/localstack/blob/master/docs/localstack-concepts/README.md#plugins) information in LocalStack. |
+| `/_localstack/init`                  | Shows the initialization status after setting up [Init hooks](https://docs.localstack.cloud/references/init-hooks/). |
 | `/_localstack/cloudformation/deploy` | Enables you to deploy CloudFormation templates locally through a web interface. |
-| `/_localstack/diagnose`| Reports extensive and sensitive data from LocalStack instance, enabled via the `DEBUG=1` configuration variable. |
+| `/_localstack/diagnose`              | Reports extensive and sensitive data from LocalStack instance, enabled via the `DEBUG=1` configuration variable. |
 | `/_localstack/config`                | Enables dynamic configuration updates at runtime, enabled via the `ENABLE_CONFIG_UPDATES` configuration variable.  |
+| `/_localstack/chaos`                 | [Chaos Plugin]({{< ref "chaos-plugin" >}}) configuration endpoint |
```

Comment:
maybe be `chaos API` is a better name here?

Reply:
Renamed to 'Chaos API' in 2f2d1932f